lib/panel/dock.js

```diff
@@ -53,11 +105,9 @@ class PanelDock {
   }
   dispose() {
     this.subscriptions.dispose()
-    const paneContainer = atom.workspace.paneContainerForItem(this)
-    if (paneContainer) {
-      if (typeof paneContainer.state === 'object' && typeof paneContainer.state.size === 'number') {
-        atom.config.set('linter-ui-default.panelHeight', paneContainer.state.size)
-      }
+    const paneContainer = getPaneContainer(this)
+    if (paneContainer && !this.alwaysTakeMinimumSpace) {
+      atom.config.set('linter-ui-default.panelHeight', paneContainer.state.size)
```

Comment:
You might want to check this against `this.panelHeight` to make sure you aren't needlessly setting it.